lib/ip_record.js

```diff
@@ -123,6 +124,32 @@ module.exports = function (limits, now) {
     this.as = this._trim(now, this.as, limits.maxAccountStatusCheck)
   }
 
+  IpRecord.prototype.isOverSMSLimit = function () {
+    this.trimSMSRequests(now())
+    // Limit based on number of unique sms request sent by this IP
+    var count = 0
+    var seen = {}
+    this.sms.forEach(function(info) {
+      if (!(info.u in seen)) {
```

Comment:
Yea I agree, it is unnecessarily hard to read. Had to use `info.u` because I was reusing the logic for [trimming](https://github.com/mozilla/fxa-customs-server/blob/606255fd5e74f2f13cf236fbf2613fcb7cb9a5dc/lib/ip_record.js#L162) hits in the record. This being said, I created the following issue https://github.com/mozilla/fxa-customs-server/issues/163 so that customs server gets some more ♥️.